### PR TITLE
Numeric selector features

### PR DESCRIPTION
**Pull request by gheorghitamutu**

### CODE REVIEW

**Review by gdt050579 — APPROVED** [review]
ok

File: AppCUI/src/Controls/NumericSelector.cpp [hunk 1]
```diff
@@ -30,47 +30,89 @@ bool NumericSelector::Create(
...
+    else
+    {
+        Members->value = value;
+    }
+}
+
+const void NumericSelector::SetMinValue(const long long minValue)
+{
+    CREATE_TYPECONTROL_CONTEXT(NumericSelectorControlContext, Members, );
+    Members->minValue = minValue;
```
Comment (gdt050579) [inline AppCUI/src/Controls/NumericSelector.cpp:60]: Trebuie validat ca o data cu setarea lui MinValue / MaxValue se pastreaza relatiile dintre ele , mai exact: minValue<=value<=maxValue. Daca relatiile nu sunt ok se adapteaza (de exemplu , daca minValue e mai mare decat maxValue, atunci se face maxValue egal cu minValue implicit (similar si pentru value)

**Review by gdt050579 — CHANGES_REQUESTED** [review]

File: AppCUI/src/Controls/NumericSelector.cpp (hunk 1 above)
Comment (gheorghitamutu) [reply AppCUI/src/Controls/NumericSelector.cpp:60]: corect

Comment (gheorghitamutu) [reply AppCUI/src/Controls/NumericSelector.cpp]: https://github.com/gdt050579/AppCUI/pull/14/commits/5c43badfa88042cd6d5669fe66c29f6a2550a779

Comment (gheorghitamutu) [reply AppCUI/src/Controls/NumericSelector.cpp:61]: https://github.com/gdt050579/AppCUI/pull/14/commits/d937f68f08384c3c8b35ccb8990b3e3305dfa968

**Review by gdt050579 — APPROVED** [review]
ok

**Pull request merged**